src/docs/platform-automation/pruning.md

```diff
@@ -0,0 +1,103 @@
+
+* Prune all builds whose build configuration no longer exists, status is complete, failed, error, or canceled.
+* Per `BuildConfig` object, keep the last 2 `Builds` whose status is `Complete`.
+* Per `BuildConfig` object, keep the last 1 `Builds` whose status is `Failed`, `Error`, or `Canceled`.
+* Per `BuildConfig` object, keep any `Builds` that is younger than 48 hours.
+
+## ImageStreams
+
+### Daily Prune
+
+As new images are built and pushed to tags, the older copies of that tag are retained. Each of these is called a revision and is tracked with a `sha256`. After some time, the number of these can pile up. So older ones are pruned daily.
+
+A few settings are passed to the pruner job:
+
+* All revisions created in the last 48h are kept
+* All revisions in use by a running pod, or in a deployment are kept. See [the docs](https://docs.openshift.com/container-platform/4.13/applications/pruning-objects.html#pruning-images-conditions_pruning-objects) for a full list
+* Only the most recent revision is kept
+* All other revisions are pruned
+
+### Image Registry Size warnings
+
+The platform will now be sending a weekly email to teams that are using too much space on the registry. The image registry is a shared service and overuse of it can lead to other teams being unable to push their builds, or to the platform team having to buy more storage space.
+
+The emails will list all the image streams, their tags, and revisions. It will show the size used by each of these ensuring to not double-count layers that are reused. This should help provide info on where to focus efforts to reduce image registry usage.
+
+You can delete a whole image stream with `oc delete imagestream <is_name>` or just a tag with `oc tag -d <imagestream>:<tag>` . See [Managing Image Streams](https://docs.openshift.com/container-platform/4.13/openshift_images/image-streams-manage.html#images-imagestream-remove-tag_image-streams-managing) for more.
+
+If you have any questions please reach out on [#devops-operations](https://chat.developer.gov.bc.ca/channel/devops-operations) and someone will be happy to help you clean up your projects image streams.
+
+The [OpenShift 201 training](/training-from-the-platform-services-team/) features content related to image stream tags. There is a related lab exercise and video demonstration [here](https://github.com/BCDevOps/devops-platform-workshops/blob/master/openshift-201/image-management.md)
+
+### Hard Prune
+
+The OpenShift Container Registry can accumulate blobs that are not referenced by the cluster’s ETCD database. The basic pruning images CronJob, therefore, is unable to operate on them. These are called orphaned blobs.
+
+To address this a hard prune of the image registry is performed during OpenShift upgrades. We complete this Sunday morning at 6am before the upgrade is started on Monday. To perform a hard prune, the image registry must first be put into a Read-Only mode to ensure data consistency.
```

Comment:
```suggestion
To address this, a robust pruning of the image registry is performed during OpenShift upgrades. We conduct this process on Sunday morning at 6 am, just before initiating the upgrade on Monday. Prior to performing a hard prune, the image registry is set to Read-Only mode to ensure data consistency.
```

Reply:
Can we stick to the "hard prune" phrase? As that's what the docs call it https://docs.openshift.com/container-platform/4.13/applications/pruning-objects.html#pruning-hard-pruning-registry_pruning-objects

The rest of the suggested changes are good with me.

Reply:
Edited again to hard prune instead